<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartOcPrinter</class>
 <widget class="QDialog" name="SmartOcPrinter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OC Printer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPrintPreviewWidget" name="previewWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="cmdZoomIn">
       <property name="text">
        <string>Zoom In</string>
       </property>
       <property name="icon">
        <iconset resource="../../universal/debtcollection/res.qrc">
         <normaloff>:/images/zoom-in.ico</normaloff>:/images/zoom-in.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cmdZoomOut">
       <property name="text">
        <string>Zoom Out</string>
       </property>
       <property name="icon">
        <iconset resource="../../universal/debtcollection/res.qrc">
         <normaloff>:/images/zoom-out.ico</normaloff>:/images/zoom-out.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cmdPDF">
       <property name="text">
        <string>    PDF   </string>
       </property>
       <property name="icon">
        <iconset resource="../../universal/smartpay/src/res.qrc">
         <normaloff>:/images/PDF.png</normaloff>:/images/PDF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="cmdPrint">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
       <property name="icon">
        <iconset resource="../../home/src/res.qrc">
         <normaloff>:/images/Printer.png</normaloff>:/images/Printer.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPrintPreviewWidget</class>
   <extends>QWidget</extends>
   <header>qprintpreviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../universal/smartpay/src/res.qrc"/>
  <include location="../../universal/debtcollection/res.qrc"/>
  <include location="../../universal/smartpay/src/res.qrc"/>
  <include location="../../home/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
